src/test/scala/uk/gov/nationalarchives/notifications/LambdaSpecUtils.scala

```diff
@@ -2,7 +2,7 @@ package uk.gov.nationalarchives.notifications
 
 import com.github.tomakehurst.wiremock.WireMockServer
 import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder
-import com.github.tomakehurst.wiremock.client.WireMock.{ok, post, urlEqualTo}
+import com.github.tomakehurst.wiremock.client.WireMock.{ok, okJson, post, urlEqualTo}
 import com.github.tomakehurst.wiremock.common.FileSource
 import com.github.tomakehurst.wiremock.core.WireMockConfiguration
 import com.github.tomakehurst.wiremock.extension.{Parameters, ResponseDefinitionTransformer}
```

Comment:
Not related to you changes but `ResponseDefinitionTransformer` looks like its deprecated. Can this be swapped to its replacement? Leave if it looks like this would be a lot of work to do.